docs/deploy.md

```diff
@@ -0,0 +1,222 @@
+# Deploying Authorino
+
+1. [Clone the repo](#clone-the-repo)
+2. [Meet the requirements](#meet-the-requirements)
+3. [Deploy](#deploy)
+   - [Option A: Local cluster](#option-a-local-cluster)
+   - [Option B: Custom deployment](#option-b-custom-deployment)
+
+## Clone the repo
+
+This is the easiest way to get Authorino's Kubernetes resources and deployment utils.
+
+```sh
+git clone git@github.com:3scale-labs/authorino.git && cd authorino
+```
+
+## Meet the requirements
+
+Having a [Kubernetes](https://kubernetes.io/) server up and running is the only actual requirement to deploy Authorino.
+
+With the tools provided in the Authorino repo nevertheless, you can easily have a local cluster setup (using [Kind](https://kind.sigs.k8s.io)). In this case, the only requirement is [Docker](https://docker.com).
```

Comment:
as in, no need for kubernetes?

Reply:
=> Nevertheless, with the tools provided in the Authorino repo…

Reply:
Correct. Kind starts a Kubernetes cluster in a docker container.